plugins/drude/openmmapi/src/DrudeLangevinIntegrator.cpp

```diff
@@ -98,6 +98,7 @@ void DrudeLangevinIntegrator::setDrudeFriction(double coeff) {
 }
 
 void DrudeLangevinIntegrator::cleanup() {
+    context = nullptr;
```

Comment:
What is this change for?  It looks unrelated to the rest of the PR.

Reply:
I think it caused some tests to fail, but i can't recall the reasoning behind this change. I reverted it, because it is indeed unrelated